src/components/data/SelectionDrawer.js

```diff
@@ -78,13 +78,11 @@ function SelectionToolbar(props) {
     // The path key includes the file name if it's a file
     const handleConfirm = () => {
         if (multiSelect) {
-            const resourceNames = selectedResources.map(
-                (resource) => resource.path
-            );
-            onConfirm(resourceNames);
+            onConfirm(selectedResources);
```

Comment:
I am returning array of disk resource object only in case of multi-select to keep the changes to minimum.